### ISSUE TITLE
Screen flickering when heavy workload (Ventura)

### ISSUE THREAD

**Issue by dinaradenis** [SuhailSherief/ThinkPad-x260-macOS-OpenCore#11]
When heavy workload, the screen would flickering, in my case when i playing game, the screen will flicker for few second, then back to normal. when just standby on the desktop it doesn't do that, is that fixable?

**Comment by SuhailSherief** [comment]
Yeah, I have the same issue with Ventura too. I guess it needs some fine tuning and I will try it. Meanwhile use Monterey build. It is stable under heavy workload and yes, I played games too in Monterey (league of legends) so go for it. Thanks a lot buddy I will keep you updated :) also support me by starring the repo and following me... bless <3

**Comment by dinaradenis** [comment]
So far the workaround is to change the screen resolution such as 1280x720

**Comment by SuhailSherief** [comment]
> So far the workaround is to change the screen resolution such as 1280x720

That's great to hear I'll pin this issue :)

**Comment by SuhailSherief** [comment]
did you find the fix @dinaradenis ?
